test/telepoison_test.exs

```diff
@@ -25,11 +25,28 @@ defmodule TelepoisonTest do
       assert_receive {:span, span(attributes: attributes_record, name: "GET")}
       attributes = elem(attributes_record, 4)
 
-      assert ["http.method", "http.status_code", "http.url", "net.peer.name"] ==
+      assert [
+               "http.method",
+               "http.request.method",
+               "http.response.status_code",
+               "http.status_code",
+               "http.url",
+               "net.peer.name",
+               "server.address",
+               "server.port",
+               "url.full",
+               "url.scheme"
+             ] ==
                attributes |> Map.keys() |> Enum.sort()
 
       assert {"http.method", "GET"} in attributes
```

Comment:
we could maybe start adding comments to attributes that should be deprecated, WDYT?

Reply:
WDYM? How?

Reply:
I was thinking about simple plain comment like
```elixir
# deprecated
```
:laughing:

Reply:
So `# deprecated` after each old convention in this list?

https://github.com/primait/telepoison/blob/b64102b2fcab26e4b618a79a61347c30d2db741b/lib/telepoison.ex#L22-L33

Reply:
yep, something like that :shrug: